Flexx/Classes/Flexx.swift

```diff
@@ -237,12 +238,11 @@ public class Flexx {
     /// Handle locale for all domains
     /// - Parameters:
     ///   - domains: list of domain names
-    ///   - locale: current locale
-    private func handleLocaleSynchronicallyForDomains(_ domains: [String], locale: Locale) {
+    ///   - localeId: current locale identifier
+    private func handleLocaleSynchronicallyForDomains(_ domains: [String], localeId: String) {
```

Comment:
I can see two usages of that func, on 128 and 181 lines.
For the first usage (128 line) we pass NOT fixed `localeId` and for the second (181 line) we pass fixed `localeId`. I suggest to pass it fixed for all its usages for consistency and to remove re-fixing the `localeId` on 282 line.